### ISSUE TITLE
Apostrophe won't display in InDesign

### ISSUE THREAD

**Issue by JRK1710014** [fontforge/designwithfontforge.com#187]
Hi,

I just finished designing a font with FontForge. I followed the steps here: http://designwithfontforge.com/en-US/Making_Sure_Your_Font_Works_Validation.html to troubleshoot it, fixed all the issues (and there were a lot, since it is the very first font I've designed), then exported it to test it out. This is where I encountered a problem: The apostrophe won't display in InDesign, I only get the little square. I ran back "Find Problems...", but it can't see anything wrong with it.
I figured it might be linked to the design itself, so I copy-pasted the apostrophe of a "real" font, but it won't display either.
It works fine in Word and other places I tried, just InDesign (And Illustrator). But since I created it in order to use it in InDesign, it is a bit problematic... It might be because my Adobe suite is a bit old now, CS2, but considering it's a font I made, it's probably my fault...
Any ideas on how to find what is wrong and fix it?
(I can't add the font file directly, so here is a link to it if it can help: https://drive.google.com/open?id=1wxX_DK49CEo1eimF0jESAKaPulsDSIKA)

Thanks in advance!

**Comment by skef** [comment]
First debugging experiment: Make a copy of the font, delete the GPOS kerning tables entirely, then export and see if you still see the problem. (The fact that the only character causing problems is the only character   as a first kerning table pair points to that as the potential problem area, so best to check that first.)

**Comment by JRK1710014** [comment]
Hi, Thanks for your reply. I tried that, to no avail, but I did figure it out: When leaving the pointer on a character for a few second a little box appears, with a list of other characters that are preferred, or can be used. I guess Windows and Adobe just don't use the same unicode character for the apostrophe. After copy-pasting the design in all the other possible apostrophes it works!
(I found that I had the same issue for the quote marks, so I did the same and it works as well)
